torch/_inductor/utils.py

```diff
@@ -244,7 +244,8 @@ def sympy_str(expr: sympy.Expr):
 
 
 def sympy_symbol(name):
-    return sympy.Symbol(name, integer=True, positive=True)
+    from torch.fx.experimental.symbolic_shapes import Symbol
+    return Symbol(name, integer=True, positive=True)
```

Comment:
I know this is hacks, but it would be really great if symbolic_shapes was the source of truth for making new Symbols

Reply:
Should be resolved.

Reply:
Jk, we can't do this for now.